f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_acquisition_package.xml

```diff
@@ -26,36 +27,39 @@
             </choice>
         </element>
         <index name="index">
-            <element name="period_of_performance"/>
+            <element name="fair_opportunity"/>
         </index>
         <index name="index2">
-            <element name="fair_opportunity"/>
+            <element name="project_overview"/>
         </index>
         <index name="index3">
-            <element name="project_overview"/>
+            <element name="gfe_overview"/>
         </index>
         <index name="index4">
-            <element name="gfe_overview"/>
+            <element name="sensitive_information"/>
         </index>
         <index name="index5">
-            <element name="contract_award"/>
+            <element name="classification_level"/>
         </index>
         <index name="index6">
-            <element name="sensitive_information"/>
+            <element name="contract_considerations"/>
         </index>
         <index name="index7">
-            <element name="classification_level"/>
+            <element name="contract_award"/>
         </index>
         <index name="index8">
             <element name="contract_type"/>
         </index>
         <index name="index9">
-            <element name="organization"/>
+            <element name="period_of_performance"/>
         </index>
         <index name="index10">
-            <element name="contract_considerations"/>
+            <element name="requirements_cost_estimate"/>
         </index>
         <index name="index11">
+            <element name="organization"/>
+        </index>
+        <index name="index12">
             <element name="current_environment"/>
```

Comment:
Is this intended behavior?

Reply:
Nevermind, it seems index change is related to adding that new table in [d7c7d15](https://github.com/dod-ccpo/atat-snow/pull/67/commits/d7c7d150142a36fbfc6fd39b8567fedd8207d33d)